src/main/java/com/imsweb/geocoder/PointInPolygonInput.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 Information Management Services, Inc.
+ */
+package com.imsweb.geocoder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PointInPolygonInput {
+
+    private Integer _censusYear;
+    private Double _lat;
+    private Double _lon;
```

Comment:
I definitely think lat and lon should be strings. We store them as strings in DMS because we had issues converted them to doubles. So we don't want to convert them to doubles and then back to strings.

I would also be OK with census year being a string to simplify things a little. Don't feel as strongly about it, though.